deepspeech_pytorch/configs/lightning_config.py

```diff
@@ -24,39 +31,41 @@ class ModelCheckpointConf:
 
 @dataclass
 class TrainerConf:
-    _target_: str = "pytorch_lightning.trainer.Trainer"
-    logger: Any = True  # Union[LightningLoggerBase, Iterable[LightningLoggerBase], bool]
+    _target_: str = "pytorch_lightning.Trainer"
+    logger: Any = (
+        True  # Union[LightningLoggerBase, Iterable[LightningLoggerBase], bool]
+    )
     enable_checkpointing: bool = True
-    callbacks: Any = None  # Optional[List[Callback]]
     default_root_dir: Optional[str] = None
     gradient_clip_val: float = 0
-    process_position: int = 0
+    callbacks: Any = field(
+        default_factory=lambda: [
+            TQDMProgressBar(process_position=0, refresh_rate=1),
+            DeviceStatsMonitor(),
+            ModelSummary(),
+            StochasticWeightAveraging(swa_lrs=1e-2),
```

Comment:
I would suggest not adding swa by default

Reply:
do we need to add any as default?

Reply:
I'm not very sure about this. @carmocca - can you help with Sean's question?

Also, once and if you both agree, I'll remove `StochasticWeightAveraging(swa_lrs=1e-2)` line from here.

Reply:
This PR was prompted by the failure in eco-ci: https://github.com/Lightning-AI/ecosystem-ci/runs/8081079930?check_suite_focus=true
Since passing the `process_position` to the Trainer was removed in favor of passing the `TQDMProgressBar`.

Config-based systems have this problem of the configs becoming outdated. The eco-ci test does not pass the `process_position`, the error appears because the configuration file includes the old argument, even though it doesn't customize its value.

So my suggestion is that we don't add any callbacks by default. Just removing the `process_position` argument should be enough to fix eco-ci

Reply:
@krshrimali i think we should just get rid of all default callbacks

Reply:
Thanks, @SeanNaren and @carmocca!

Just a minor correction:

> Just removing the process_position argument should be enough to fix eco-ci

We need to remove other outdated arguments as well, as it wasn't just `process_position`, but other arguments as well that have been removed in PL 1.7.0. 

I've added the commit, please take a look :)